packages/react-dom/rollup.config.js

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+import plugin from "@eik/rollup-plugin";
+import { createRequire } from "module";
+
+const { resolve } = createRequire(import.meta.url);
+const reactPkg = new URL("../react/package.json", import.meta.url);
+const reactDomPkg = new URL("./package.json", import.meta.url);
```

Comment:
This is super sweet, just discovered it in the Node docs. 

```js
const { pathname } = new URL("./", import.meta.url);
// pathname here is the same as _dirname
```

Reply:
Niceness. Was not aware of this one.